src/style/Provider.tsx

```diff
@@ -0,0 +1,83 @@
+/** @jsx jsx */
+import React, { Fragment } from 'react'
+import { ScreenClassProvider, setConfiguration } from 'react-grid-system'
+import { css, Global, jsx, ThemeProvider, useTheme } from './emotion'
+import { ThemeProps } from './theme'
+import { NeedsChildrenProps } from './util'
+
+const Children: React.FC = ({ children }) => {
+  const theme = useTheme()
+  setConfiguration(theme.grid.rules)
+  return (
+    <Fragment>
+      <Global styles={theme.global.css(theme)} />
+      <Global styles={theme.font.text.css(theme)} />
+      <Global
+        styles={css`
+          h1,
+          h2,
+          h3,
+          h4,
+          h5,
+          h6 {
+            ${theme.font.h1h6.css(theme)}
+          }
+        `}
+      />
+      <Global
```

Comment:
We don't need all the global sections for fonts.  Instead, I would just take a `cssCss` argument for where we can implement whatever we want for font (or anything else).  Basically, I don't think giving font any special structure is helpful.